tensorflow/java/BUILD

```diff
@@ -111,6 +111,11 @@ cc_binary(
             LINKER_EXPORTED_SYMBOLS,
         ],
         "//tensorflow:windows": [],
+        "//tensorflow:debug" : [
```

Comment:
I agree that it would have been nice, but unfortunately nesting `select`s doesn't quite work out well (https://github.com/bazelbuild/bazel/issues/1623)

@karllessard : `-z defs` doesn't work on all platforms (the linker in OS X for example). How about we leave the set empty for debug builds. So, add `"//tensorflow:debug": [],` as the first condition in the `select` statement here so that for debug builds we don't do any custom linker configuration.